exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `greeting`
+let greeting = "hello world";
```

Comment:
Well Done Adiba, I am trying to use const instead of let in my code as well for those variable which I am sure that will not change. As TA's at class mention about it :)